benchmark/events/ee-listener-count.js

```diff
@@ -1,7 +1,7 @@
 var common = require('../common.js');
 var EventEmitter = require('events').EventEmitter;
 
-var bench = common.createBenchmark(main, {n: [25e4]});
+var bench = common.createBenchmark(main, {n: [5e7]});
```

Comment:
Some complete more quickly than others.  This particular benchmark is the quickest of them all.